api/v1/flow/flow.proto

```diff
@@ -156,6 +159,16 @@ message Layer7 {
     }
 }
 
+// Trace contains information about a distributed trace.
+message Trace {
+    TraceContext context = 1;
+}
+
+// TraceContext is metadata about trace spans.
+message TraceContext {
```

Comment:
Why is `TraceContext` a separate message, instead of having the `trace_id` field directly in `Trace`? Do you have plans for extending it?

Reply:
Second thought: having separate `TraceContext` is convenient if we want to support different propagation formats than w3c. I don't foresee that, but I guess it's good to have that separation.

Reply:
Ok. I expect other formats to remain in the wild for quite some time, but the incentive for supporting them in Hubble should be low.

Reply:
> I expect other formats to remain in the wild for quite some time

Yes, probably. However, I think we don't have to support all older tracing implementations in Hubble itself as it would complicate trace information extraction and the Hubble flow definition for no real gain. As explained above, legacy systems may still parse HTTP headers directly as a last resort option.